snippets/dart_snippets/lib/connecting_lsp.dart

```diff
@@ -10,6 +10,12 @@ Future<void> getLspInfo() async {
   // ANCHOR_END: get-lsp-info
 }
 
+Future<void> listLsps() async {
+  // ANCHOR: list-lsps
+  List<LspInformation> availableLsps = await BreezSDK().listLsps();
+  // ANCHOR_END: list-lsps
+}
```

Comment:
return availableLsps after ANCHOR_END change fn type to List<LspInformation> to pass CI

Reply:
I'm guessing same applies to Swift example.

Reply:
Done